source/install/desktop-managed-resources.rst

```diff
@@ -0,0 +1,38 @@
+
+Desktop - Managed Resources
+===========================
+
+This guide provides steps to use managed resources with the Mattermost Desktop App. A managed resource can be any service available on the same hostname using the same protocol as the Mattermost server.
+
+.. note::
+    Using this feature requires a `custom build of the Mattermost Desktop App <https://docs.mattermost.com/deployment/desktop-app-deployment.html>`_.
+
+When adding the folder of a managed resource to `src/common/config/buildConfig.js <https://github.com/mattermost/desktop/blob/master/src/common/config/buildConfig.js>`_ it is available from the Mattermost Desktop App as a pop-up window instead of opening in a web-browser. In the below example we add the managed resource `/video`.
+
+.. code-block:: json
+    [...]
+    managedResources: ['trusted', 'video'],
+    [...]
+
+Here are two example server URLs each with valid and invalid managed resource URLs:
```

Comment:
Suggestion: Include an introduction or sentence around the resource URLs (it's not immediately clear how they're related to the example).

Reply:
Another option might be details around what makes the URL valid/invalid (looks like it's because it needs to be HTTPS).

Reply:
I've changed `folder` to `path`, would that make more sense now?

Reply:
I can't see where that it - is it part of the example?

Reply:
Is that in this example, or for the line "When adding the path of a managed resource to"?

Reply:
@Willyfrog, should we keep the examples below or should we trim them down to an example of one valid and one invalid?

Reply:
I think those are fine, but if it feels like too much info they can be removed. I'm 0/5 on that.